systemd/openqa-scheduler.service

```diff
@@ -1,7 +1,6 @@
 [Unit]
 Description=The openQA Scheduler
-Before=openqa-webui.service openqa-websockets.service
```

Comment:
IIRC we saw later that this was a problem after we tested it, or did we fixed it already in master?

Reply:
I don't know - I've seen cycle warnings, but we need to get this correct. Perhaps @lnussel can review